docs/integration.md

```diff
@@ -0,0 +1,248 @@
+# [Integration Guide](#integration-guide)
+The basis of this document is to chowcase the process of integrating `bridget` into an existing web delivery. The end-goal of this module is adding bridging support to the website when browsed through the context of a mobile app via a WebView.
+
+## [Prerequisites](#prerequisites)
+
+This integration guide assumes you already have a web delivery that serves content & a mobile app is being setup for that  particular web delivery.
+
+## [Requirements](#requirements)
+
+#TODO - write requirements that the web delivery needs to satisfy in order for the module to work
+
+## [How to install](#how-to-install)
+> The module is designed to run in the browser. It can also be used in a server (node) environment via npm.
+
+### [Installation via script tag](#install-via-script-tag)
+```html
+<script src="https://unpkg.com/@forward-distribution/bridget">
+  // The module will bootstrap itself when imported, if it is in the correct context (mobile webview)
+</script>
+```
+Put the above script tags in between the <head> tags of your HTML page. It will attach a global `bridget` object. You can access it either via `bridget` or `window.bridget`.
+
+## [Alternative installation via NPM](#alternative-installation-via-npm)
+The module is available on NPM, if you’re using a front-end packager like Browserify or Webpack:
+
+Just run
+
+```
+npm install -g @forward-distribution/bridget
+```
+
+You can then `require` or `import` the lib like any other module.
+
+## [Internals](#internal)
```

Comment:
Internals? 
We are trying to make "public api" and give this to integrators... really, internals doesn't feel right at all